sematic/api/endpoints/runs.py

```diff
@@ -498,3 +499,11 @@ def link_resource_endpoint(user: Optional[User], run_id: str) -> flask.Response:
     external_resource_ids = flask.request.json["external_resource_ids"]
     save_run_external_resource_links(resource_ids=external_resource_ids, run_id=run_id)
     return flask.jsonify({})
+
+
+@sematic_api.route("/api/v1/runs/<run_id>/metrics", methods=["GET"])
```

Comment:
Please put this under some other endpoint, in order to leave "metrics" reserved for the upcoming fleshed-out feature, and have an independent deprecation process for this feature?
How about "basic_metrics" to mirror the Dashboard code?

Reply:
The endpoint in the other PR is `/api/v1/metrics` with filter arguments. Since metrics can be queried across many different dimensions.